classes/04class/exercises/c04-iac03/starkmatt/README.md

```diff
@@ -0,0 +1,14 @@
+# C04-IAC03
+
+## Command Execution Output
+- [output.txt](output.txt)
+
+## Questions:
+- Describe below why some changes just updated the resource being changed and the other required a full replacement of the resource.
+    - The VPC Cidr / Subnet Cidr changed without any issues because the change was made on the variable value itself, hence the data would be called the same way initally.
```

Comment:
can you really change a VPC cidr in place? Without deleting and creating a new one?

Reply:
Hey Caio,
This is a good point, it didn't really make sense how we can have a CIDR that was different from the subnets.
Although when I ran a terraform plan, it didn't show up with any errors.
Would the error come during the apply phase ?

Reply:
the terraform plan only checks syntax and if you are linking resources correctly.. the apply should complain though

Out of interest, you can add more secondary CIDR's to a vpc but it's best to always plan ahead rather than having to do changes along the way. 

https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Subnets.html#vpc-resize